simplified-viewer-epub-readium1/src/main/java/org/nypl/simplified/viewer/epub/readium1/ReaderActivity.java

```diff
@@ -510,9 +510,7 @@ private void onProfileEventPreferencesChanged(
   }
 
   private void applyReaderPreferences(final ReaderPreferences preferences) {
-    LOG.debug("applyReaderPreferences: scheduling task to run later");
-
-    uiThread.runOnUIThreadDelayed(() -> {
+    uiThread.runOnUIThread(() -> {
```

Comment:
I'm removing the execution delay here. This function is now called from `onReadiumContentDocumentLoaded`, so a delay is no longer necessary to wait for the content to load. It also continues to be called from `onProfileEventPreferencesChanged`, which happens when the user selects a new font/color scheme. I believe the delay was not needed in this case, and was only making the app less responsive.